# Request message types
---
# Response message types
float64 pixel_height
float64 pixel_width
